[2023-08-29T19:43:56.304+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.316+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.336+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-29T19:43:56.337+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-29T19:43:56.351+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-29T19:43:56.361+0000] {standard_task_runner.py:57} INFO - Started process 358 to run task
[2023-08-29T19:43:56.369+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpnw_l3qzl']
[2023-08-29T19:43:56.374+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask import_data_to_csv
[2023-08-29T19:43:56.433+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.504+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host e69093e17de8
[2023-08-29T19:43:56.526+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-29T19:43:56.626+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-29T19:43:56.634+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-29T19:43:56.637+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-29T19:43:56.654+0000] {subprocess.py:86} INFO - Output:
[2023-08-29T19:43:56.815+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-29T19:43:56.822+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-29T19:43:56.840+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-29T19:43:56.846+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230829T194356, end_date=20230829T194356
[2023-08-29T19:43:56.875+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 20 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 358)
[2023-08-29T19:43:56.914+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-29T19:43:56.939+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T18:56:46.293+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.305+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.319+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.321+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:46.340+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T18:56:46.348+0000] {standard_task_runner.py:57} INFO - Started process 547 to run task
[2023-08-30T18:56:46.355+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpqyq__xpt']
[2023-08-30T18:56:46.359+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T18:56:46.383+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.440+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:46.465+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.568+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T18:56:46.571+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:46.573+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T18:56:46.588+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:46.690+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T18:56:46.693+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T18:56:46.706+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T18:56:46.713+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T185646, end_date=20230830T185646
[2023-08-30T18:56:46.742+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 10 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 547)
[2023-08-30T18:56:46.764+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T18:56:46.787+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:20:00.549+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.561+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.572+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.575+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:20:00.599+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:20:00.611+0000] {standard_task_runner.py:57} INFO - Started process 271 to run task
[2023-08-30T19:20:00.612+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmps5jjt99x']
[2023-08-30T19:20:00.617+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask import_data_to_csv
[2023-08-30T19:20:00.636+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.685+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:20:00.712+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.790+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:20:00.794+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:20:00.798+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python ./dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T19:20:00.829+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:20:00.943+0000] {subprocess.py:93} INFO - python: can't open file './dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T19:20:00.953+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T19:20:00.971+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T19:20:00.978+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T192000, end_date=20230830T192000
[2023-08-30T19:20:00.994+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 8 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 271)
[2023-08-30T19:20:01.036+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T19:20:01.058+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:37.205+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.218+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.230+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.232+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:37.252+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:31:37.262+0000] {standard_task_runner.py:57} INFO - Started process 269 to run task
[2023-08-30T19:31:37.266+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpfvjhka_x']
[2023-08-30T19:31:37.270+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T19:31:37.298+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.343+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:37.365+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.450+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:31:37.456+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:37.460+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T19:31:37.475+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:41.022+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:41.059+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T193137, end_date=20230830T193141
[2023-08-30T19:31:41.090+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:41.115+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:53.565+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.578+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.591+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.592+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:53.607+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:41:53.612+0000] {standard_task_runner.py:57} INFO - Started process 326 to run task
[2023-08-30T19:41:53.616+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpp_8pv4cz']
[2023-08-30T19:41:53.620+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask import_data_to_csv
[2023-08-30T19:41:53.644+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.700+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:53.726+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.825+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:41:53.829+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:53.830+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T19:41:53.851+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:59.749+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:59.784+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T194153, end_date=20230830T194159
[2023-08-30T19:41:59.825+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:59.850+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:53.719+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.736+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.757+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.759+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:53.785+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:50:53.793+0000] {standard_task_runner.py:57} INFO - Started process 307 to run task
[2023-08-30T19:50:53.797+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp0nc3ut3a']
[2023-08-30T19:50:53.801+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T19:50:53.826+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.894+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:53.914+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:54.001+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:50:54.003+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:54.006+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T19:50:54.020+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:51:01.734+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:51:01.819+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T195053, end_date=20230830T195101
[2023-08-30T19:51:01.866+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:51:01.899+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:26.254+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.264+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.276+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:26.290+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:54:26.296+0000] {standard_task_runner.py:57} INFO - Started process 258 to run task
[2023-08-30T19:54:26.303+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpx7jn877x']
[2023-08-30T19:54:26.307+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T19:54:26.328+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.388+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:26.415+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.511+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:54:26.514+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:26.515+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T19:54:26.527+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:30.511+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:30.545+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T195426, end_date=20230830T195430
[2023-08-30T19:54:30.578+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:30.615+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:49.242+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.254+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.264+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.267+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:49.292+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:05:49.299+0000] {standard_task_runner.py:57} INFO - Started process 469 to run task
[2023-08-30T20:05:49.304+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpud67rrb6']
[2023-08-30T20:05:49.308+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask import_data_to_csv
[2023-08-30T20:05:49.324+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.389+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:49.410+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.498+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:05:49.500+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:49.502+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T20:05:49.515+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:56.544+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:56.643+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T200549, end_date=20230830T200556
[2023-08-30T20:05:56.715+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:56.763+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:50.110+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.124+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.136+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.137+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:50.163+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:33:50.176+0000] {standard_task_runner.py:57} INFO - Started process 287 to run task
[2023-08-30T20:33:50.184+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmprx9q6psi']
[2023-08-30T20:33:50.189+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask import_data_to_csv
[2023-08-30T20:33:50.211+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.260+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:50.279+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.365+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:33:50.367+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:50.369+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T20:33:50.386+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:56.276+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:56.311+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T203350, end_date=20230830T203356
[2023-08-30T20:33:56.355+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:56.388+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:23.525+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.540+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.549+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.550+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:23.566+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:48:23.574+0000] {standard_task_runner.py:57} INFO - Started process 339 to run task
[2023-08-30T20:48:23.582+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpthhahwwf']
[2023-08-30T20:48:23.586+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T20:48:23.606+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.655+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:23.687+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.760+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:48:23.762+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:23.763+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T20:48:23.778+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:30.684+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:30.747+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T204823, end_date=20230830T204830
[2023-08-30T20:48:30.786+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:30.818+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:36.655+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.671+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.689+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.694+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:36.714+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T21:07:36.720+0000] {standard_task_runner.py:57} INFO - Started process 299 to run task
[2023-08-30T21:07:36.726+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpablzxtu_']
[2023-08-30T21:07:36.731+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T21:07:36.777+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.903+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:36.931+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:37.068+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T21:07:37.074+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:37.078+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T21:07:37.104+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:52.763+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:52.865+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T210736, end_date=20230830T210752
[2023-08-30T21:07:52.917+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:52.960+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:17:04.213+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.229+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.240+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.243+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:17:04.276+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-28 00:00:00+00:00
[2023-08-30T21:17:04.284+0000] {standard_task_runner.py:57} INFO - Started process 357 to run task
[2023-08-30T21:17:04.299+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpptrhoc6w']
[2023-08-30T21:17:04.306+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T21:17:04.337+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.407+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-28T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:17:04.429+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.524+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T21:17:04.527+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:17:04.528+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-28']
[2023-08-30T21:17:04.543+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:17:12.636+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:12.668+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230828T000000, start_date=20230830T211704, end_date=20230830T211712
[2023-08-30T21:17:12.728+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:12.749+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
